src/std/main.ab

```diff
@@ -266,3 +266,44 @@ pub fun shell_var_get(name: Text): Text {
 pub fun shell_unset(name: Text): Null {
     $unset {name}$?
 }
+
+pub fun shell_text(message: Text, color1: Num, style: Num, color2: Num): Text {
+    return "\e[{style as Text};{color1 as Text};{color2 as Text}m{message}\e[0m"
+}
+
+pub fun bold_text(message: Text): Text {
+    return "\e[1m{message}\e[0m"
+}
+
+pub fun italic_text(message: Text): Text {
+    return "\e[3m{message}\e[0m"
+}
+
+pub fun underlined_text(message: Text): Text {
+    return "\e[4m{message}\e[0m"
+}
+
+pub fun color_echo(message: Text, color: Num): Null {
+    unsafe $echo -e "\e[{color as Text}m{message}\e[0m"$
+}
+
+pub fun echo_esc(message: Text): Null {
+    unsafe $echo -e "{message}"$
+}
+
+pub fun info(message: Text): Null {
+    unsafe $echo -e "\e[1;3;97;44m {message} \e[0m"$
+}
+
+pub fun success(message: Text): Null {
+    unsafe $echo -e "\e[1;3;97;42m {message} \e[0m"$
+}
+
+pub fun warning(message: Text): Null {
+    unsafe $echo -e "\e[1;3;97;43m {message} \e[0m"$
+}
+
+pub fun error(message: Text, exit_code: Num = 1): Null {
+    unsafe $echo -e "\e[1;3;97;41m {message} \e[0m"$
+    if exit_code >= 0 : exit(exit_code)
+}
```

Comment:
Can we add a prefix to the names to `print_` so that it makes more sense?

`print_info`
`print_warning`
`print_error`
`print_success`